Programan/jaime_ramirez_madrigal/Ejercicio1.js

```diff
@@ -0,0 +1,11 @@
+
+let suma = 0;
+let ingreso = 0;
+
+for (let i = 5; i > 0; i--) {

```

Comment:
El uso de `--` y `++` tiene mal rendimiento, se recomienda utilizar `i += 1` , `i = i +1`, `i -= 1` , `i = i - 1`.

Juega con este código para ver el performance

> let i = 0;
console.time("i++");
for (let j = 0; j < 9999999; j+=1) {
    i+=1;
}
console.timeEnd("i++");